### DISCUSSION TITLE
Using Google Compute VM local storage or Firebase storage instead of S3 or DigitalOcean

### DISCUSSION THREAD

**Discussion by LeftoversTodayAppAdmin** [OpenSignLabs/OpenSign#392] [General]
Hey folks!
Where can I find instructions to use Firebase or just store locally on the Google Compute VM? 
I am not on AWS or DigitalOcean. Any help here will be appreciated!
Thank you
Shri

**Comment by chourdeau** [comment]
Hello,

Have you found a solution to your question? Were you able to move forward with your deployment? 

I'd probably be interested in testing the solution and I'm also on Google Cloud Plateform (GCP).

Thanks for your feedback

Cédric

**Reply by wmeneses** [reply]
Hi, I use GCP I have deployed docker on one of my hosting services with plesk, but on a normal Vm it should work the same, I was able to save in bucket both signatures (images) and pdfs with the same amazon s3 instructions. but the documents cannot be visulized, because of the CROS access.
